### PR TITLE
Fixes before release

### PR DESCRIPTION
**Pull request by jmysliv**
## Description

Many fixes before release. PR including fixes for react-native-screens: https://github.com/software-mansion/react-native-reanimated/pull/2340

Authors: @Szymon20000 @piaskowyk @jmysliv

### CODE REVIEW

**Comment by BooYeu** [comment]
When can we use 2.2.1 ? We keep waiting in hope~~~Thank you very much~~~

**Review by piaskowyk — APPROVED** [review]

**Comment by piaskowyk** [comment]
@BooYeu After release 2.3.0-alpha.3 I will work on the release of 2.2.1 with support for RN65

File: Common/cpp/Tools/Scheduler.cpp [hunk 1]
```diff
@@ -12,8 +12,11 @@ void Scheduler::scheduleOnJS(std::function<void()> job) {
 }
 
 void Scheduler::triggerUI() {
-  auto job = uiJobs.pop();
-  job();
+  scheduledOnUI = false;
+  while (uiJobs.getSize()) {
+    auto job = uiJobs.pop();
+    job();
```
Comment (pistonsky) [inline Common/cpp/Tools/Scheduler.cpp:18]: Can we wrap this line into try-catch? People are getting crashes on this line.

**Review by pistonsky — COMMENTED** [review]
`Scheduler::triggerUI` needs to be more stable!

**Pull request merged**